.github/actions/azure-functions-integration-setup/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Azure Functions Integration Test Setup
```

Comment:
Seems like we can probably reuse this for the .NET integration tests too, though currently we're just setting it up inline: https://github.com/microsoft/agent-framework/pull/2070.

One thing I might do differently is call it something slightly different since even for non-Azure Functions integration tests we still need DTS.